ee/tabby-ui/lib/experimental-flags.ts

```diff
@@ -0,0 +1,78 @@
+import useLocalStorage from 'use-local-storage'
+
+type DefineExperimentalFlagResponse = [
+  ExperimentalFlag,
+  () => {
```

Comment:
return array of flag and toggle function

Reply:
updated, the return type is 
```js
[
  { value: boolean, description: string },
  () => void
]
```